### DISCUSSION TITLE
Upload to Gdrive is not working.

### DISCUSSION THREAD

**Discussion by Meliodas-Demonking** [anasty17/mirror-leech-telegram-bot#1156] [Q&A]
Help me with this issue please 

Pic: https://te.legra.ph/file/7bf65454bcf5aa103aded.jpg

**Comment by anasty17** [comment]
Token.pickle corrupted

**Reply by Meliodas-Demonking** [reply]
Corrupted how? is it because I changed client secret and id?

**Reply by anasty17** [reply]
Seems because u done something wrong ! That's it!
